### PR TITLE
chore: add MilkyWay glados-2.1 details

### PR DESCRIPTION
**Pull request by RiccardoM**
## Description

This PR adds the details for the `glados-2.1` MilkyWay chain

### CODE REVIEW

File: testnets/milkyway/chain.json [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+      "png": "https://raw.githubusercontent.com/initia-labs/initia-registry/main/testnets/milkyway/images/MilkyWay.png",
+      "svg": "https://raw.githubusercontent.com/initia-labs/initia-registry/main/testnets/milkyway/images/MilkyWay.svg"
+    }
+  ],
+  "logo_URIs": {
+    "png": "https://raw.githubusercontent.com/initia-labs/initia-registry/main/testnets/milkyway/images/MilkyWay.png",
+    "svg": "https://raw.githubusercontent.com/initia-labs/initia-registry/main/testnets/milkyway/images/MilkyWay.svg"
+  },
+  "metadata": {
+    "op_bridge_id": "98",
```
Comment (ALPAC-4) [inline testnets/milkyway/chain.json:78]: please recheck the bridge id. https://lcd.testnet.milkyway.zone/opinit/opchild/v1/bridge_info

File: testnets/milkyway/chain.json [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+        "version": "ics20-1"
+      },
+      {
+        "chain_id": "initiation-1",
+        "port_id": "wasm.init1wug8sewp6cedgkmrmvhl3lf3tulagm9hnvy8p0rppz9yjw0g4wtq7947m6",
+        "channel_id": "channel-1",
+        "version": "ics721-1"
+      },
+      {
+        "chain_id": "initiation-1",
```
Comment (ALPAC-4) [inline testnets/milkyway/chain.json:94]: can you check the version? I think it is ics27-1

File: testnets/milkyway/chain.json [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+  "logo_URIs": {
+    "png": "https://raw.githubusercontent.com/initia-labs/initia-registry/main/testnets/milkyway/images/MilkyWay.png",
+    "svg": "https://raw.githubusercontent.com/initia-labs/initia-registry/main/testnets/milkyway/images/MilkyWay.svg"
+  },
+  "metadata": {
+    "op_bridge_id": "98",
+    "op_denoms": [],
+    "ibc_channels": [
+      {
+        "chain_id": "initiation-1",
```
Comment (ALPAC-4) [inline testnets/milkyway/chain.json:82]: can you add counterparty channels on testnets/initia/chain.json

**Comment by ALPAC-4** [comment]
And also can you add skip apis like you did on [glados-1 ](https://github.com/initia-labs/initia-registry/blob/c3f97878ec864778b32187aac64b100246b7f98d/testnets/milkyway/chain.json#L45-L49)

File: testnets/milkyway/chain.json (hunk 2 above)
Comment (RiccardoM) [reply testnets/milkyway/chain.json:94]: Do you know how to check this?

**Comment by RiccardoM** [comment]
@ALPAC-4 I have fixed the bridge id. Can I fix the other things in a PR? We need this merged ASAP as our testnet application and wallet do not work at all unless we have these things in the chain registry.

**Review by ALPAC-4 — APPROVED** [review]

**Pull request merged**